component/storage/indexeddb/indexeddb.go

```diff
@@ -79,18 +96,64 @@ func (p *Provider) OpenStore(name string) (storage.Store, error) {
 		return nil, err
 	}
 
-	return &store{name: name, db: p.stores[name]}, nil
+	return p.stores[name], nil
 }
 
-// SetStoreConfig is currently not implemented. It always returns a nil error so that IndexedDB can still be used as
-// a cache provider in FormatProvider for the time being.
+// SetStoreConfig sets the configuration on a store.
+// TODO (#2540): Use proper IndexedDB indexing instead of the "Tag Map" once aries-framework-go is updated to use the
+// new storage interface. The tag names used in the predefined stores (see getStoreNames()) will need to be passed in
+// somehow.
 func (p *Provider) SetStoreConfig(name string, config storage.StoreConfiguration) error {
```

Comment:
I am out of the loop here.

Is this method to be invoked by the framework or the user? (my guess is the framework)

Reply:
Is this method supposed to be called only once per store in its lifetime?

Reply:
> Is this method to be invoked by the framework or the user? (my guess is the framework)

Framework does it.

> Is this method supposed to be called only once per store in its lifetime?

It'll be called right after a store is created (that needs to use tags).

Reply:
> Framework does it.

How can a user assign arbitrary tags?

Reply:
> > Framework does it.
> 
> How can a user assign arbitrary tags?

Yep - although the framework will be using specific tags for is purposes. Someone extending the framework may have their own uses for tags.

Reply:
> > Framework does it.
> 
> How can a user assign arbitrary tags?

Sorry, just realized you were asking "how".

Tags are assigned via "Put" calls to associate some data as a part of some larger "grouping". Does that answer your question?